resources/Materials/Examples/StandardSurface/standard_surface_default.mtlx

```diff
@@ -21,7 +21,7 @@
       <bindinput name="transmission_extra_roughness" type="float" value="0"/>
       <bindinput name="subsurface" type="float" value="0"/>
       <bindinput name="subsurface_color" type="color3" value="1, 1, 1"/>
-      <bindinput name="subsurface_radius" type="vector3" value="1, 1, 1"/>
+      <bindinput name="subsurface_radius" type="color3" value="1, 1, 1"/>
```

Comment:
I believe the desire was to make this a color to match up more with the user presentation of this which as a color which could be scaled to derive the final distance. The pbr node internally is still a vector and a distance. Pinging @niklasharrysson for this.

Reply:
My main concern is that color inputs will be transformed between color spaces as if they were perceptual colors (e.g. radiance, reflectance), which may not behave as expected for physical distances like subsurface radius.

If this has already been discussed in detail, I don't want to slow down the process of integrating the change to MaterialX, but I wanted to double-check that this behavior is truly intended.

Reply:
Yes this change was a request from the Arnold team who is responsible for the Standard Surface specification. Their motivation for changing this to a color3 was that the perceived effect of changing it is very "color like". Which also includes connecting it to color textures that should be effected by color management. So the radius/color input can be use to control the perceptual color of the SSS and _scale_ input can then be used to turn this into a physical distance for mean free path.

As Bernard pointed out we choose to keep the parameterization of the `subsurface_bsdf` node in pbrlib with a vector3 still, because here we don't use a color + scalar but a single vector for radius. We also figured it would make sense to keep the lower level nodes in pbrlib more strictly physical while higher level nodes like `standard_surface` can choose to add more artistic interpretations of this.

Reply:
Got it, and thanks for the detailed answer, @niklasharrysson.  That makes sense to me, where the mean free path is the combination of a physical scale and a color in a particular color space.  I'll mark this question as resolved.

Reply:
While the perception of what subsurface_radius does may be color-like, it is most definitely not a color, and really should not be a color3 type for all the reasons Jonathan already mentioned.  Also, it is fairly common to multiply the subsurface_radius by a scalar, which makes total sense for a vector, but not as much so for a color.  The only reason to express subsurface_radius as a color3 I can think of is if that's the only 3-channel data type your UI supports, but in that case I would hope that the application's interface to MaterialX would already have a color3 UI -> MaterialX vector3 conversion under-the-hood, which should apply here as well.

Reply:
Thought about this more on the drive into work, and it's a bit more complex than that.. technically speaking, subsurface_radius *is* a distance (mean free path length for scattering or something like that), and is typically specified in millimeters, but it's the distance/path length *per color channel*, which is dependent on what your color primaries are.  So the questions is, if you have a subsurface_radius value relative to one set of primaries, is the conversion of that value to a different set of primaries actually using the same math as converting an actual color value?  My gut feeling is "no" but I'd defer to someone with more knowledge in this area.

Reply:
@dbsmythe I agree this is complex, and I think there are good arguments for both using a color and using a vector for this radius. The change we did now was to make the MaterialX nodedef in line with the Arnold teams description in the Standard Surface whitepaper. The Arnold team had a long debate about this internally and finally settled for using a color in the whitepaper. So I think it's best if we use the same in the MaterialX nodedef and avoid deviating from the whitepaper. In fact, as discussed before, we are planning to move this nodedef from the MaterialX repo and into the Standard Surface whitepaper repo, so it will be owned by the Arnold team. And we will just reference this in as a submodule in MaterialX.

On the other hand for the <subsurface_bsdf> nodedef in the PRB library, which is an interface that we "own" in the MaterialX repo, we are keeping the vector for radius because in that case it's a mean free path input directly, without any scalar multiplier to make it a "distance" mean free path.